package.json

```diff
@@ -8,10 +8,11 @@
     "prod": "pnpm run migrate && tsx --env-file-if-exists=.env --dns-result-order=ipv6first bin/server.ts",
     "dev": "pnpm run migrate && tsx watch --env-file-if-exists=.env --dns-result-order=ipv6first bin/server.ts",
     "list:routes": "tsx --env-file-if-exists=.env bin/routes.ts",
-    "test": "pnpm run migrate:test && tsx --env-file-if-exists=.env.test --test",
-    "test:ci": "tsx --test --test-reporter=@reporters/github --test-reporter-destination=stdout --test-reporter=spec --test-reporter-destination=stdout",
+    "test": "pnpm run migrate:test && tsx --env-file-if-exists=.env.test --test --test-concurrency=1",
+    "test:ci": "tsx --test --test-reporter=@reporters/github --test-reporter-destination=stdout --test-reporter=spec --test-reporter-destination=stdout --test-concurrency=1",
```

Comment:
We needed to switch to concurrency of 1 for now, as the database tests clean the database, which causes issues if multiple tests are running concurrently.

Reply:
Maybe in the future we could like provision the database based on test concurrency or something? I don't know. Running all the tests takes only 15s, so I'm not too phased about limiting concurrency to 1.